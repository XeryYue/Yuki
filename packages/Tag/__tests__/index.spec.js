import Tag from '../index'
import { mount } from '@vue/test-utils'

describe('Tag', () => {
  it('should be render as element', () => {
    const wrapper = mount(Tag, {
      props: {
        text: 'Test',
      },
    })
    expect(wrapper.html()).toMatchSnapshot()
    expect(() => wrapper.unmount()).not.toThrow()
  })
})
