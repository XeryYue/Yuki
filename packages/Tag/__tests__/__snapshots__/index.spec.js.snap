// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tag should be render as element 1`] = `<div class="fect-tag">Test</div>`;
